libbeat/common/event.go

```diff
@@ -33,8 +33,6 @@ import (
 
 const eventDebugSelector = "event"
```

Comment:
Do we need this const any more? Looks like it's only used in the one place now, post-refactoring.